scripts/tasks_retrival/HPA_tasks_creation.py

```diff
@@ -14,35 +18,20 @@ def import_data(url):
     return data
 
 
-def load_yaml_file(yaml_path):
-    with open(yaml_path) as f:
-        loaded_yaml = safe_load(f)
-    return loaded_yaml
-
-
-def format_pathology_columns(data):
+def format_hpa_columns(data, clear_semicolon=None):
     pathology_columns = list(
         filter(lambda x: "Pathology prognostics" in x, data.columns)
     )
     data[pathology_columns] = data[pathology_columns].replace(
         r"\s*\(\d+\.?\d*e?-?\d*\)", "", regex=True
     )
+    if not clear_semicolon is None and CELL_LINE in data.columns:
+        data[clear_semicolon] = (
+            data[clear_semicolon].astype(str).apply(lambda x: x.replace(";", ""))
+        )
```

Comment:
isnt this supposed to be `data[CELL_LINE]` ? 
whats this input? `clear_semicolon` a bool? add args types to header